tsconfig.json

```diff
@@ -0,0 +1,81 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                         /* Enable incremental compilation */
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
why select ES5? why not a later version?

Reply:
ES5 is just the default for typescript compilation. With Node 14 we can go as modern as ES2020 and still maintain 100% feature coverage. We'd need to update to Node16 to get coverage of ES2021 functionality. I've updated the target to ES2020 (the most recent ES version we can cover with Node 14, which we use for our docker image).

Reply:
You would otherwise want to target an ES version most compatible with the web browser versions you're targeting when compiling TypeScript applications that are meant to be ran in the browser (as opposed to a Node JS application).

Reply:
On second thought, I'm going to revert us to ES5 (maximum compatibility) because we still serve up a front-end application via this image. Once we have a separate frontend and this application doesn't need to serve anything runnable to a browser we can update our target to a more recent spec.